rs/ledger_suite/icrc1/tests/upgrade_downgrade.rs

```diff
@@ -67,22 +67,23 @@ fn should_upgrade_and_downgrade_ledger_canister_suite() {
     )
     .unwrap();
 
-    env.advance_time(Duration::from_secs(60));
-    env.tick();
+    // Downgrade to mainnet is not possible for this version.
+    // env.advance_time(Duration::from_secs(60));
+    // env.tick();
 
-    env.upgrade_canister(
-        index_id,
-        index_ng_mainnet_wasm(),
-        Encode!(&index_upgrade_arg).unwrap(),
-    )
-    .unwrap();
+    // env.upgrade_canister(
+    //     index_id,
+    //     index_ng_mainnet_wasm(),
+    //     Encode!(&index_upgrade_arg).unwrap(),
+    // )
+    // .unwrap();
 
-    env.upgrade_canister(
-        ledger_id,
-        ledger_mainnet_wasm(),
-        Encode!(&ledger_upgrade_arg).unwrap(),
-    )
-    .unwrap();
+    // env.upgrade_canister(
+    //     ledger_id,
+    //     ledger_mainnet_wasm(),
+    //     Encode!(&ledger_upgrade_arg).unwrap(),
+    // )
+    // .unwrap();
```

Comment:
Instead of just commenting this out, could we assert that the downgrade fails?
After V3 has been deployed to mainnet and V4 hasn't yet been merged to master, this should again succeed, right (similarly for V4 and V5)?

Reply:
Done!